### PR TITLE
Dev/370 Handle white list IP in the exam

### PR DESCRIPTION
**Pull request by Wally5077**
resolve: #370

### CODE REVIEW

File: services.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ services:
       MYSQL_HOST: "${MYSQL_HOST}"
       RABBIT_HOST: "${RABBIT_HOST}"
       REDIS_HOST: "${REDIS_HOST}"
+      _JAVA_OPTIONS: "-Djava.net.preferIPv4Stack=true"
```
Comment (Chaoyuuu) [inline services.yml:46]: Does `java.net.preferIPv4Stack: true` work?

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review Done!

File: services.yml (hunk 1 above)
Comment (Wally5077) [reply services.yml:46]: Yes, It did.

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**